apps/survey/src/components/layouts/meal-list/meal-item.vue

```diff
@@ -19,12 +19,14 @@
         </v-tooltip>
       </v-list-item-action>
       <v-list-item-action class="my-auto">
-        <context-menu
-          :entity="meal"
-          :entity-name="mealName"
-          v-bind="{ menu }"
-          @action="action"
-        ></context-menu>
+        <v-expand-transition>
```

Comment:
not sure if needed yet as like this it won't do much yet

Reply:
Thanks. This doesn't work and I will delete them.